uds/translator/data_record_definitions/did.py

```
@@ -278,6 +278,96 @@ def _get_did_data(did: int) -> Tuple[RawDataRecord]:
+            mask_data_records.append(_get_mask_data_record(dr))
+        return (RawDataRecord(name=name,
+                              children=mask_data_records,
+                              length=total_length,
+                              min_occurrences=0 if optional else 1,
+                              max_occurrences=1),)
+
+    return ConditionalFormulaDataRecord(formula=_get_did_data_mask,
+                                        default_message_continuation=[default_did_data_mask])
+
+
+def get_did_data_mask_2020(name: str, optional: bool) -> ConditionalFormulaDataRecord:
```

Ruff (0.14.4)

Boolean-typed positional argument in function definition

(FBT001)